safehttp/mux.go

```diff
@@ -96,7 +96,8 @@ func NewServeMux(d Dispatcher, domains ...string) *ServeMux {
 	}
 }
 
-type appliedInterceptor struct {
+// AppliedInterceptor TODO
+type AppliedInterceptor struct {
```

Comment:
Why is this part of the public API now?

Reply:
It was part of the public API since we wanted to pass it as part of the `ResponseWriter` constructor and for that it needs to be exported, so that the constructor doesn't become unusable outside the package.